templates/esp-crdb-ca.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
@ScottGarman Are we treating this chart as open source? If so, we should either make this chart configurable to look for an external secret that is configurable via the values file and remove our `esp`'s since they're a custom internal implementation or at a minimum we probably want to leave a note in the README like we do here: https://github.com/equinixmetal-helm/k8s-jetstack-secure/blob/main/README.md